workspace_host/interface.js

```diff
@@ -187,7 +187,7 @@ async.series([
 
 // For detecting file changes
 var update_queue = {};  // key: path of file on local, value: action ('update' or 'remove').
-const workspacePrefix = process.env.HOST_JOBS_DIR ? '/jobs/workspaces' : config.workspaceJobsDirectory;
+const workspacePrefix = config.workspaceJobsDirectory;
```

Comment:
Can we do something similar for: https://github.com/PrairieLearn/PrairieLearn/blob/b6cb352938d23f22dd14df5ea9e12605364d839d/workspace_host/interface.js#L34

Reply:
Done. Working on my end (dockerized).